pkg/ccrm/ccrm_cloudlet.go

```diff
@@ -143,36 +112,64 @@ func (s *CCRMHandler) createCloudlet(ctx context.Context, in *edgeproto.Cloudlet
 	if err != nil {
 		return err
 	}
-	return nil
-}
+	if in.CrmOnEdge {
+		// CRM will set up rootLB and capture resources
+		return nil
+	}
 
-func (s *CCRMHandler) deleteCloudlet(ctx context.Context, in *edgeproto.Cloudlet, cloudletPlatform platform.Platform, cb edgeproto.CacheUpdateCallback) error {
+	log.SpanLog(ctx, log.DebugLevelApi, "ccrm setup new cloudlet", "cloudlet", in)
+	// replicate here what CRM would have done to set up initial cloudlet state
+	// Note that "cloudletPlatform" is initialized in CreateCloudlet, similar to
+	// but not the same as platform.InitCommon(), so we need to get a different
+	// copy of the platform object that has been initialized with InitCommon.
+	pf, err := s.getCRMCloudletPlatform(ctx, &in.Key)
+	if err != nil {
+		if errors.Is(err, ErrPlatformNotFound) {
+			return nil
```

Comment:
Should we log that the platform is of unknown type?

Reply:
yup, I'll add a log